### PR TITLE
fix: Minor docstring and snippet fixes

### PR DESCRIPTION
**Pull request by vinnysenthil**
Fixes https://github.com/googleapis/python-aiplatform/issues/474, https://github.com/googleapis/python-aiplatform/issues/467, [b/196053040](http://b/196053040) 🦕

### CODE REVIEW

**Review by dizcology — APPROVED** [review]

File: samples/snippets/prediction_service/predict_text_classification_single_label_sample.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def predict_text_classification_single_label_sample(
     )
     print("response")
     print(" deployed_model_id:", response.deployed_model_id)
-    # See gs://google-cloud-aiplatform/schema/predict/prediction/text_classification.yaml for the format of the predictions.
+
```
Comment (dandhlee) [inline samples/snippets/prediction_service/predict_text_classification_single_label_sample.py:48]: Missing updated comment here to point to new storage blob

**Review by dandhlee — CHANGES_REQUESTED** [review]

File: samples/snippets/prediction_service/predict_text_classification_single_label_sample.py (hunk 1 above)
Comment (vinnysenthil) [reply samples/snippets/prediction_service/predict_text_classification_single_label_sample.py:48]: There is no YAML prediction output schema for text classification in the bucket, so I went ahead and removed the comment altogether:
```shell
gsutil ls gs://google-cloud-aiplatform/schema/predict/prediction/
```

Comment (dandhlee) [reply samples/snippets/prediction_service/predict_text_classification_single_label_sample.py:48]: Got it!

**Review by dandhlee — APPROVED** [review]
LGTM.

**Review by sasha-gitg — APPROVED** [review]

**Pull request merged**